openflow13/action.go

```diff
@@ -276,7 +276,7 @@ func (a *ActionGroup) UnmarshalBinary(data []byte) error {
 type ActionMplsTtl struct {
 	ActionHeader
 	MplsTtl uint8
-	pad     []byte // 3bytes
+	// 3 bytes for padding
```

Comment:
isn't this comment a bit confusing here, since there is nothing in the struct enforcing that padding?